src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -14,6 +14,7 @@ export default Object.freeze({
   vaOnlineSchedulingPast: 'vaOnlineSchedulingPast',
   vaGlobalDowntimeNotification: 'vaGlobalDowntimeNotification',
   ssoe: 'ssoe',
+  ssoeInbound: 'ssoeInbound',
```

Comment:
@hinzed1127 I think this needs to be `ssoe_inbund` to match https://github.com/department-of-veterans-affairs/vets-api/pull/4202

Reply:
it looks like the flipper controller [converts it](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/feature_toggles_controller.rb#L14).

The client [docs](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-toggles/#writing-good-feature-toggles) say to do camelCase, probably just going by language conventions in each

Reply:
oh cool, I had no idea

Reply:
The API uses a gem called olive_branch, which lets caller request conversion of JSON keys to camelCase based on the value of the "X-Key-Inflection" (or something like that) header. Was done to allow Node frontend and Rails backend to have as-expected inflection in their respective code domains.